tests/regionizers/test_administrative_boundary_regionizer.py

```diff
@@ -57,35 +59,40 @@ def test_no_crs_gdf_value_error(gdf_no_crs) -> None:  # type: ignore
         abr.transform(gdf=gdf_no_crs)
 
 
-@pytest.mark.parametrize(  # type: ignore
-    "toposimplify",
-    [
-        (True),
-        (0.0001),
-        (0.001),
-        (0.01),
-        (False),
-        (0),
-    ],
-)
-def test_single_points(toposimplify: Union[bool, float]) -> None:
-    """Test checks if single points work."""
-    country_points_gdf = gpd.GeoDataFrame(
+@pytest.fixture  # type: ignore
+def mock_for_madagascar(mocker: MockerFixture) -> None:
+    """Mock overpass API."""
+    mocker.patch.object(
+        API,
+        "get",
+        return_value={
+            "elements": [
+                {
+                    "type": "relation",
+                    "id": 2137,
+                },
+            ]
+        },
+    )
+
+    geocoded_gdf = gpd.GeoDataFrame(
         {
             "geometry": [
-                Point(19.24530, 52.21614),  # Poland
-                Point(10.48674, 51.38001),  # Germany
-                Point(14.74938, 47.69628),  # Austria
-                Point(15.00989, 49.79905),  # Czechia
-            ]
+                box(
+                    minx=47,
+                    miny=-15,
+                    maxx=48,
+                    maxy=-14,
+                )
+            ],
         },
         crs=WGS84_CRS,
     )
-    abr = AdministrativeBoundaryRegionizer(
-        admin_level=2, return_empty_region=False, clip_regions=False, toposimplify=toposimplify
+
+    mocker.patch(
+        "osmnx.geocode_to_gdf",
+        return_value=geocoded_gdf,
     )
-    countries_result_gdf = abr.transform(gdf=country_points_gdf)
-    assert list(countries_result_gdf.index) == ["Poland", "Germany", "Austria", "Czechia"]
```

Comment:
We should change this test to test a single point and ensure it's inside returned geometry. Now we don't cover points path in code coverage. Or maybe resort to unit tests and check some private functions inside a class individually.

Reply:
I added this test - please take a look to see if it's ok, or maybe I went too far with mocking